cmd/boulder-va/gsb_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestConfigCheck(t *testing.T) {
```

Comment:
I tried to write a test for `NewGoogleSafeBrowsingv4` but it seems like instantiating a `safebrowsingv4.NewSafeBrowser` causes it to try and validate the API key works against a server. Since `NewGoogleSafeBrowsingv4` is 90% `configCheck` and 10% calling `safebrowsingv4.NewSafeBrowser` and putting it in the adapter type it isn't worth writing a test server for the unit test. I think we can skip a `new...` test here.